Resources/Prototypes/_StarLight/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml

```
@@ -0,0 +1,42 @@
+  - type: Appearance
+  - type: ItemSlots
+    slots:
+      gun_magazine:
+        name: Magazine
+        startingItem: SmartgunMagazineLightRifleBox
+        insertSound: /Audio/Weapons/Guns/MagIn/batrifle_magin.ogg
+        ejectSound: /Audio/Weapons/Guns/MagOut/batrifle_magout.ogg
+        priority: 2
+        whitelist:
+          tags:
+            - LightRifleHeavy
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)